addons/point_of_sale/tests/test_anglo_saxon.py

```diff
@@ -287,13 +287,21 @@ def test_cogs_with_ship_later_no_invoicing(self):
         aml_output = aml.filtered(lambda l: l.account_id.id == account_output.id)
         aml_expense = aml.filtered(lambda l: l.account_id.id == expense_account.id)
 
+        self.assertEqual(len(aml_output), 3, "There should be 3 output account move lines")
+        # 2 moves in POS journal (Pos order + manual entry at delivery)
+        self.assertEqual(aml_output[0:1].move_id.journal_id, self.pos_config.journal_id)
```

Comment:
```suggestion
        self.assertEqual(aml_output[0:2].move_id.journal_id, self.pos_config.journal_id)
```
Do you intend to get the first 2 move lines? Then the index should be `0:2` or `:2`.